arangod/IResearch/IResearchInvertedIndex.cpp

```diff
@@ -1038,6 +1058,11 @@ void IResearchInvertedClusterIndex::toVelocyPack(
   builder.add(arangodb::StaticStrings::IndexName, velocypack::Value(name()));
   builder.add(arangodb::StaticStrings::IndexUnique, VPackValue(unique()));
   builder.add(arangodb::StaticStrings::IndexSparse, VPackValue(sparse()));
+
+  if (isOutOfSync()) {
+    // link is out of sync - we need to report that
+    builder.add(StaticStrings::LinkOutOfSync, VPackValue(true));
```

Comment:
Given we plan to handle `broken` indexes as well maybe we have to stick to enumeration, e.g.
```
error: "outOfSync"
```

Reply:
done